docs/modules/ROOT/pages/release-notes/release-notes-2021-6.adoc

```diff
@@ -0,0 +1,29 @@
+= Release notes - Payara Platform Development - Version 5.2021.6
```

Comment:
Please capitalize Notes.  and change Development to Community, like this:
Release Notes - Payara Platform Community 5.2021.6